### DISCUSSION TITLE
Menu item with same name

### DISCUSSION THREAD

**Discussion by cedricve** [thuliteio/doks#326] [Q&A]
When having a pages with the same name, across multiple first level section it will not be shown (on one of the two places).

    ---
    title: "First things first"
    description: ""
    lead: ""
    date: 2020-10-06T08:49:31+00:00
    lastmod: 2020-10-06T08:49:31+00:00
    draft: false
    images: []
    menu:
    docs:
    parent: "enterprise"
    weight: 300
    toc: true
    ---
    
    
    
    ---
    title: "First things first"
    description: ""
    lead: ""
    date: 2020-10-06T08:49:31+00:00
    lastmod: 2020-10-06T08:49:31+00:00
    draft: false
    images: []
    menu:
    docs:
    parent: "opensource"
    weight: 300
    toc: true
    ---

**Comment by h-enk** [comment]
@cedricve, see https://github.com/h-enk/doks/discussions/117

**Comment by creio** [comment]
Custom title name sidebar menu, ~~title~~?

**Reply by h-enk** [reply]
Update `docs`, part of `Site.Menu.docs` in https://github.com/h-enk/doks/blob/master/layouts/partials/sidebar/docs-menu.html

Will be made easier in a next release

**Reply by creio** [reply]
```
---
title: Lorem
menu:
  docs:
    title: Custom title
    parent: "help"
weight: 10
---
```

**Reply by h-enk** [reply]
Ah, I see what you mean now. I'll get back to you later.

**Reply by creio** [reply]
Well, how to do that?

```
{{ $menutitle := where .Site.Pages "Section" "docs" }}
{{ $data := index .Site.Menus.docs }}

{{ range $menutitle }}
  {{ .Params.menutitle }}
{{ end }}
```
concate?

**Reply by h-enk** [reply]
With page frontmatter (for example):

```md
title: "Introduction"
..
menu:
  docs:
    parent: "prologue"
    title: "Introz"
weight: 100
..
```

In `./layouts/partials/sidebar/docs-menu.html`, on line 17, replace `{{ .Name }}` with `{{ .Title }}`:

```html
<li><a class="docs-link rounded{{ if $active }} active{{ end }}" href="{{ .URL | relURL }}">{{ .Title }}</a></li>
```
![Snag_17cb07b6](https://user-images.githubusercontent.com/3902872/137860537-08027171-7ebb-43d7-8804-98407c6d8bd9.png)
